### PR TITLE
Launcher English strings review

### PR DESCRIPTION
**Pull request by krs0**
@IvanSavenko Only sources changed, lupdate.exe needs to be run again!

### CODE REVIEW

File: launcher/firstLaunch/firstlaunch_moc.ui [hunk 1]
```diff
@@ -413,7 +413,7 @@ Heroes III: HD Edition is currently not supported</string>
             </sizepolicy>
            </property>
            <property name="text">
-            <string>Alternatively, you can select directory with installed Heroes III data and VCMI will copy exisiting data automatically.</string>
+            <string>Alternatively, you can provide the directory where Heroes III data is installed and VCMI will automatically copy the existing data.</string>
```
Comment (kambala-decapitator) [inline launcher/firstLaunch/firstlaunch_moc.ui:416]:
```suggestion
            <string>Alternatively, you can provide the directory where Heroes III data is installed and VCMI will copy the existing data automatically.</string>
```

File: launcher/firstLaunch/firstlaunch_moc.ui [hunk 2]
```diff
@@ -482,7 +482,7 @@ Heroes III: HD Edition is currently not supported</string>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="labelDataFailure">
            <property name="text">
-            <string>Automatic detection of language failed. Please select language of your Heroes III copy</string>
+            <string>The automatic detection of the Heroes III language has failed. Please manually select the language of your Heroes III</string>
```
Comment (kambala-decapitator) [inline launcher/firstLaunch/firstlaunch_moc.ui:485]:
```suggestion
            <string>The automatic detection of the Heroes III language has failed. Please select the language of your Heroes III manually.</string>
```

File: launcher/firstLaunch/firstlaunch_moc.ui [hunk 3]
```diff
@@ -804,7 +804,7 @@ Heroes III: HD Edition is currently not supported</string>
             </sizepolicy>
            </property>
            <property name="text">
-            <string>Install compatible version of addon Horn of the Abyss: fan-made Heroes III expansion, ported by VCMI team</string>
+            <string>Install compatible version of Horn of the Abyss, a fan-made Heroes III expansion ported by the VCMI team</string>
```
Comment (kambala-decapitator) [inline launcher/firstLaunch/firstlaunch_moc.ui:807]:
```suggestion
            <string>Install compatible version of &quot;Horn of the Abyss&quot;, a fan-made Heroes III expansion ported by the VCMI team</string>
```

**Review by kambala-decapitator — APPROVED** [review]
I guess you'll need to run `lupdate` again if you accept any suggestion

**Review by IvanSavenko — APPROVED** [review]
Looks OK for me with kambala suggestions. Thanks.

**Comment by krs0** [comment]
Updated as per review

**Pull request merged**